resdk/resources/utils.py

```diff
@@ -114,3 +114,95 @@ def get_process_id(process):
 def get_sample_id(sample):
     """Return id attribute of the object if it is sample, othervise return given value."""
     return sample.id if type(sample).__name__ == 'Sample' else sample
+
+
+def get_relation_id(relation):
+    """Return id attribute of the object if it is relation, othervise return given value."""
+    return relation.id if type(relation).__name__ == 'Relation' else relation
+
+
+def is_collection(collection):
+    """Return ``True`` if passed object is Collection and ``False`` otherwise."""
+    return type(collection).__name__ == 'Collection'
```

Comment:
Because you cannot import `Collection` in to this file. Otherwise it causes cyclic import. It was your advice last time to do it this way :)